src/qubitmaps.jl

```diff
@@ -0,0 +1,76 @@
+"""
+    jordanwigner(H::OpSum; threshold = 1e-15)
+
+JORDAN-WIGNER TRANSFORMATION
+
+aⱼ  = - (∏ᵢ₌₁ʲ  Zᵢ) (Xᵢ + iYᵢ)/2
+aⱼ† = - (∏ᵢ₌₁ʲ  Zᵢ) (Xᵢ - iYᵢ)/2
+"""
+function jordanwigner(H::OpSum; cutoff = 1e-20)
```

Comment:
I would vote for using the keyword argument `atol` here instead of `cutoff`, as discussed above.